pkgs/by-name/ta/tacent/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Can you remove `rec` and use `finalAttrs` instead?
`rec` causes very unintuitive overriding semantics.

Reply:
This shouldn't be a request change as it brings little improvements to this derivation, but is more stylistic change

Reply:
The usage of rec like this causes several issues when using overrideAttrs on the derivation, so I think it's better to avoid introducing new instances of this pattern, and instead use finalAttrs. I agree it's not necessarily blocking, but especially for new packages I think we should aim to not perpetuate these historic footguns.

Reply:
See complaints like https://github.com/NixOS/nixpkgs/issues/310373.

Reply:
Uhh so should I use finalAttrs or rec?

Reply:
There is no hard requirement on using finalAttrs over rec. Do as you wish.